app/services/courses/copy_to_provider_service.rb

```diff
@@ -65,6 +69,10 @@ def copy_schools(course:, new_provider:, new_course:)
       end
     end
 
+    def course_start_is_same_as_current_cycle_start?(course)
+      course.applications_open_from == current_cycle.application_start_date
```

Comment:
`current_cycle` is `RecruitmentCycle.current_recruitment_cycle` which is then set in `Settings` right?

Would it be possible to use `CycleTimetable.current_year` which would align us with Apply?

Could we then use the [apply_reopens](https://github.com/DFE-Digital/publish-teacher-training/blob/main/app/services/find/cycle_timetable.rb#L101) method to set the date?

Appreciate I might not fully grasp the complexities of Publish course dates but happy to chat!

Reply:
Yes that’s right.

It is possible, but it will add a lot of complexity and I’m not sure the benefits we gain from it will be worth it. The date is set manually in the cycle time table, the same way it is set in the recruitment cycle. 

What’s stopping us from going all the way and setting the `application_start_date` from the cycle time table when we create the recruitment cycle? That way we only need to keep track of and update the one date in the cycle time table. Even if we do this we would still need to consider rolling over courses where the `applications_open_from` date is different to the `application_start_date` as we would want to retain these.

My gut would be to merge this as it solves the problem at hand and add a card to the backlog to re-think the approach when we have more time but also happy to continue working on it.

Reply:
Had a chat offline and decided to use the cycle switcher for now and consider alternative joined up approaches between the services in the future.